### ISSUE TITLE
Unabandoned solution back in Mentor Queue

### ISSUE THREAD

**Issue by F3PiX** [exercism/exercism#4657]
A solution turned up in the Mentoring Queue on Ruby unexpectedly. 

It has an ongoing discussion. 
At the moment of discovery the last interaction had been ~5 days before. 
I doublechecked with the mentor: they didn't leave the conversation and are waiting for a new submission by the student. 

UUID: :  994b082546e34b57a2a17471dffd3d0d

**Comment by iHiD** [comment]
Thanks for reporting.
This was caused by https://github.com/exercism/website/pull/542 which didn't deploy properly because of https://github.com/exercism/website/commit/3e78733cd5752d655f78123eee7776070bea2207

It's worth noting that the mentor didn't click the "no action required" button so this would have correctly timed out 48hrs later. 

I've added the mentor back to this solution and (effectively) clicked that button for them.
